core/test/Cardano/Address/Style/ShelleySpec.hs

```diff
@@ -1235,6 +583,18 @@ prop_roundtripTextEncodingPointer encode' decode addXPub ptr discrimination =
     address = pointerAddress discrimination (PaymentFromKey addXPub) ptr
     result  = decode (encode' address)
 
+goldenText :: String -> Text -> Golden Text
+goldenText name actualOutput =
```

Comment:
```suggestion
goldenText :: Text -> TL.Text -> Golden TL.Text
goldenText name output = Golden
```
May as well keep using lazy text, if that's what comes out of the pretty printer.

Since you have RecordWildCards enabled, you can use it for the output field.

And the whitespace police request that you observe existing conventions and put braces and commas at the start of record field lines, not the end.

Reply:
It kinda also wants `NamedFieldPuns`, but I'm a bit less for overdoing it.